content/en/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Ingress (Kubernetes)
+description: Describes how to configure a Kubernetes Ingress object to expose a service outside of the service mesh.
+weight: 15
+keywords: [traffic-management,ingress]
+---
+
+This task describes how to configure Istio to expose a service outside of the service mesh cluster, using the Kubernetes [Ingress Resource](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+
+{{< tip >}}
+Using the [Istio Gateway](/docs/tasks/traffic-management/ingress/ingress-control/), rather than Ingress, is recommended to make use of the full feature set that Istio offers.
+{{< /tip >}}
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
```

Comment:
Instead of a copy of the same instructions as the Ingress Gateways task, in this section and next, why not just say:
```
## Before you begin

Follow the instructions in the [Before you begin](link) and [Determining the ingress IP and ports](link) sections of the [Ingress Gateways task]().
```
```